sea-config.json

```diff
@@ -1,5 +1,9 @@
 {
   "main": "./build/p0.js",
   "output": "./build/p0.blob",
-  "disableExperimentalSEAWarning": true
+  "disableExperimentalSEAWarning": true,
+  "assets": {
+    "favicon.ico": "./public/favicon.ico",
+    "redirect-landing.html": "./public/redirect-landing.html"
```

Comment:
Unfortunately I don't think there's a way to do this without having to explicitly enumerate the static assets we want to include individually. These are used as the landing webpage post-login.